### DISCUSSION TITLE
Fancybox Zoom buttons?

### DISCUSSION THREAD

**Discussion by wlsquid** [fancyapps/ui#58] [Q&A] [answered]
I can see in the docs that there are zoom buttons for panzoom but I was wondering if these zoom controls could be put on fancybox (like those on panzoom) as well?

If not there is way to call the zoom functions through custom buttons for fancybox?
![image](https://user-images.githubusercontent.com/57164151/129141154-ae803464-4474-4238-a19a-cce5bc27fb82.png)

**Comment by fancyapps** [comment, marked as answer]
Hi,

This way you can access the current slide and the Panzoom instance created by the image plugin - 

```JS
Fancybox.getInstance().getSlide().Panzoom
```
You can then call any Panzoom method (zoomIn()/zoomOut()/toggleZoom()), example:

```JS
Fancybox.getInstance().getSlide().Panzoom.toggleZoom();
```

Those buttons are created by Controls plugin of the Panzoom and it is not included by default. Simply include appropriate CSS and JS files and then register the plugin -

```
Panzoom.Plugins = {
  Controls,
};
```

Demo - https://fancyapps.com/playground/zY

But I think you might better want to add zoom in/out buttons in the toolbar, see this demo - https://fancyapps.com/playground/zZ

**Reply by wlsquid** [reply]
Hi,

Thanks for the reply. This helps a lot. Will give these a go.
